src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -253,6 +254,7 @@ <h3>[[localize('settings-metrics-header')]]</h3>
         this.$.keysPortCancelButton.disabled = !enabled;
       },
       _handleKeysPortClick: function() {
+        if (!this.isAccessKeyPortEditable) { return; }
```

Comment:
nit: maybe drop the return to the next line

Reply:
Done